eZ/Publish/API/Repository/Tests/LocationServiceTest.php

```diff
@@ -2597,17 +2598,78 @@ public function testMoveSubtreeHidden()
         // Load new parent location
         $newParentLocation = $locationService->loadLocation($demoDesignLocationId);
 
+        // Move location from "Home" to "Demo Design" (not container)
+        $locationService->moveSubtree($locationToMove, $newParentLocation);
+    }
+
+    /**
+     * Test for the moveSubtree() method.
+     *
+     * @see \eZ\Publish\API\Repository\LocationService::moveSubtree()
+     * @depends eZ\Publish\API\Repository\Tests\LocationServiceTest::testLoadLocation
+     * @expectedException \eZ\Publish\API\Repository\Exceptions\InvalidArgumentException
+     */
+    public function testMoveSubtreeThrowsExceptionOnMoveToSame()
```

Comment:
At least for all new test cases (methods), ideally for all the affected since this should not pollute the diff too much.
```suggestion
    public function testMoveSubtreeThrowsExceptionOnMoveToSame(): void
```